frontend/lib/src/features/user_profile/profile_change_password.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+import 'package:frontend/src/components/auth/auth_header.dart';
+import 'package:frontend/src/navigators/profile_screen_navigator.dart';
+import 'package:frontend/src/components/breadcrumb.dart';
+import 'package:frontend/src/components/input/input_field.dart';
+import 'package:frontend/src/components/button.dart';
+
+/*
+  The page where user can change their password.
+*/
+class ProfileChangePassword extends StatelessWidget {
+  const ProfileChangePassword({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormBuilderState>();
+
+    return GestureDetector(
+      onTap: () => FocusManager.instance.primaryFocus?.unfocus(),
+      child: Scaffold(
+        body: SingleChildScrollView(
+          child: Column(
+            children: [
+              const AuthHeader(hasAuthToken: true),
+              Breadcrumb(
+                text: 'Change Password',
+                onTap: () {
+                  return profileAppNav.currentState?.popUntil((route) {
+                    return route.isFirst;
+                  });
+                },
+              ),
+              Padding(
+                padding: const EdgeInsets.fromLTRB(25, 21, 25, 150),
+                child: FormBuilder(
+                  key: formKey,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      const InputField(
+                        name: 'old_password',
+                        labelText: 'Old Password',
+                        obscureText: true,
+                      ),
+                      const SizedBox(
+                        height: 26,
+                      ),
+                      const InputField(
+                        name: 'new_password',
+                        labelText: 'New Password',
+                        obscureText: true,
+                      ),
+                      const SizedBox(
+                        height: 26,
+                      ),
+                      const InputField(
+                        name: 'confirm_new_password',
```

Comment:
request,

add TODO: checkout password confirmation mechanics of laravel, they are using some naming conventions